apps/prairielearn/src/components/QuestionContainer.html.ts

```diff
@@ -655,19 +659,45 @@ function QuestionPanel({
             questionNumber: instance_question_info?.question_number,
           })}
         </h1>
-        ${showCopyQuestionButton
-          ? html`
-              <button
-                class="btn btn-light btn-sm ms-auto"
-                type="button"
-                data-bs-toggle="modal"
-                data-bs-target="#copyQuestionModal"
-              >
-                <i class="fa fa-clone"></i>
-                Copy question
-              </button>
-            `
-          : ''}
+        <div class="ms-auto d-flex flex-row gap-1">
+          <div class="btn-group">
+            ${showCopyQuestionButton
+              ? html`
+                  <button
+                    class="btn btn-sm btn-outline-light"
+                    type="button"
+                    data-bs-toggle="modal tooltip"
+                    data-bs-title="Copy question"
+                    data-bs-target="#copyQuestionModal"
+                  >
+                    <i class="fa fa-fw fa-clone"></i>
+                  </button>
+                `
+              : ''}
+            ${manualGradingPreviewUrl
+              ? html`
+                  <div class="btn-group">
+                    <button
+                      class="btn btn-sm btn-outline-light dropdown-toggle dropdown-toggle-no-caret"
+                      type="button"
+                      aria-expanded="false"
+                      data-bs-toggle="dropdown tooltip"
+                      data-bs-title="Preview..."
+                    >
+                      <i class="fa fa-fw fa-eye"></i>
+                    </button>
+                    <ul class="dropdown-menu dropdown-menu-end">
+                      <li>
+                        <a class="dropdown-item" href="${manualGradingPreviewUrl}">
+                          Preview manual grading view
```

Comment:
Am I understanding correctly that we'll later add another option in this menu to preview the AI grading view?